language-support/scala/codegen/src/main/scala/com/digitalasset/codegen/Util.scala

```diff
@@ -32,6 +32,8 @@ abstract class Util(val packageName: String, val outputDir: File) { self =>
 
   type RefType = TypeConNameOrPrimType
 
+  val roots: Seq[Regex] = Nil
```

Comment:
If you do still need to tell `Util` this regex list, this `val` should be abstract.